third_party/go/BUILD

```diff
@@ -12,6 +12,8 @@ java_library(
         "intellij-ue-2018.1": ["@go_2018_1//:go"],
         "intellij-2018.2": ["@go_2018_2//:go"],
         "intellij-ue-2018.2": ["@go_2018_2//:go"],
+        "intellij-2018.3": ["@go_2018_2//:go"],
+        "intellij-ue-2018.3": ["@go_2018_2//:go"],
```

Comment:
You didn't update the third party references to the plugins, that's going to break support for those languages if the plugins are incompatible. I'm going to test some out and report back.

Reply:
Ok, at least for scala it still works, since we don't have any golang code yet in the main repo I think it's ok.

Reply:
Yup. I just ignored go. Should've explained sorry.